sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.getApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -63,7 +64,7 @@ describe("ApplicationOfferingChangeRequestAESTController(e2e)-getApplicationOffe
           applicationOfferingChangeRequest.application.location.institution
             .operatingName,
         submittedDate: applicationOfferingChangeRequest.createdAt.toISOString(),
-        updatedDate: applicationOfferingChangeRequest.updatedAt.toISOString(),
+        studentActionDate: null,
```

Comment:
IMO. it would be better, if you could specify somewhere, maybe in the test description that the application offering change request is in progress with the student